source/layouts/partials/_solo_boinc_intro.erb

```diff
@@ -0,0 +1,49 @@
+<section class="boinc-a grc-bgcolor grclink pt-4" data-background-color="black">
+    <!--Start of GRC Feature section-->
+    <!--First row of features-->
+    <div class="container">
+        <div class="row">
+            <div class="col-xs-12 col-sm-8">
+                <h5>Not interested in earning BOINC rewards?</h5>
+                <p>Switch to the <a href="/Guides/investor-gridcoin-setup.htm">Investor guide</a>. You do not need to install or run BOINC in order to use GRC as a cryptocurrency.</p>
+                <h4 class="text-bold">BOINC Introduction</h4>
+                <p>In order to earn more Gridcoin, you need to crunch one or more whitelisted <a href="https://boinc.berkeley.edu/">BOINC</a> projects within team Gridcoin. You must install the <a href="https://boinc.berkeley.edu/">BOINC</a> client, select several <a href="https://boinc.berkeley.edu/">BOINC</a> projects and get crunching!</p>
+                <p>To retrieve your crunching rewards by solo mining, you must have a balance of Gridcoin sufficient to stake before 180 days elapses. To be safe you should have enough Gridcoin to stake approximately once every month.</p>
+                <p> Averaging statistics over the last month, this would be around 5,000 GRC. GRC can be acquired in a number of different ways <a href="/exchange.htm">here.</a> You can also start out by <a href="/Guides/pool.htm">pool mining</a>, which requires no initial investment of Gridcoin to retrieve your rewards.</p>
+                <h5>BOINC video introductions:</h5>
+                <ul>
+                    <li><a href="https://www.youtube.com/watch?v=MHiUninQn18">David Anderson talks BOINC, Citizen Science, and Why We Still Need You!</a></li>
+                    <li><a href="https://www.youtube.com/watch?v=--Vy5ee7DwE">Serge Berig - BOINC: Compute for Good, Change the World</a></li>
+                </ul>
+                
+            </div>
+            <div class="col-sm-4 hidden-xs">
+                <a href="https://boinc.berkeley.edu/"><img src="/assets/img/boinc_logo2.png" alt="BOINC Logo" /></a>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-xs-12">
+                <h4 class="text-bold">Install BOINC</h4>
+            </div>
+        </div>
+
+        <!--Step 1-->
+        <div class="row">
+            <div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">
+                <p>Navigate to the <a href="https://boinc.berkeley.edu">BOINC website</a>, download and install the appropriate BOINC client on each of the computers you wish to crunch BOINC work upon. If you're not crunching projects which require <a href="https://www.virtualbox.org/wiki/Downloadsl">VirtualBox</a> you don't need to install BOINC+VirtualBox.</p>
```

Comment:
link should be `https://www.virtualbox.org/wiki/Downloads` without the "l" at the end.